src/components/Layout.tsx

```diff
@@ -10,28 +10,28 @@ export default function Layout({ children }: LayoutProps) {
     return (
         <div className="bg-[url('/fire-bg.jpg')]">
             <div className="bg-[url('/fire-sil.png')] bg-right-bottom bg-no-repeat bg-fixed bg-[length:310px_233px] flex flex-col">
-                <header className="header">
+                <header className="flex justify-between h-[100px] shadow-[0_5px_black]">
                     <div
                         onMouseEnter={() => {
                             setIsOpen(true);
                         }}
                         onMouseLeave={() => {
                             setIsOpen(false);
                         }}
-                        className="pokeball"
+                        className="flex items-start h-full bg-[url('/pixelpokeball.png')] bg-[0_0] [aspect-ratio:75/100] hover:animate-pokeballopening"
```

Comment:
Just a hint: `bg-[0_0]` is usually the same as `bg-left-top`. But in this case it's ok to use 0_0 because you are manipulating the position on a pixel base in the animation.

Nothing to be done here. Only a heads-up.